src/components/Sidebar/components/Aggregator/Aggregator.spec.js

```diff
@@ -148,6 +163,18 @@ describe('Aggregator', () => {
 
       expect(aggregator.state().open).toBe(false);
     });
+
+    it('should not render children when disabled', () => {
+      const wrapper = mount(
+        <Aggregator {...props}>
```

Comment:
Did you mean to use `disabledProps` here?
```suggestion
        <Aggregator {...disabledProps}>
```

Reply:
Yes!